test/geocoders/algolia.py

```diff
@@ -0,0 +1,76 @@
+import unittest
+
+from geopy.geocoders import AlgoliaPlaces
+from test.geocoders.tomtom import BaseTomTomTestCase
+from test.geocoders.util import GeocoderTestBase, env
+
+
+@unittest.skipUnless(
+    bool(env.get('ALGOLIA_PLACES_APP_ID')) and bool(env.get('ALGOLIA_PLACES_API_KEY')),
+    'No ALGOLIA_PLACES_APP_ID and/or no ALGOLIA_PLACES_API_KEY env variables setted'
+)
+class AlgoliaPlacesTestCase(BaseTomTomTestCase, GeocoderTestBase):
```

Comment:
The Geocoder itself doesn't extend TomTom, but tests do -- that looks strange. Moreover, some tests from `BaseTomTomTestCase` (like `test_reverse`) are overridden, which screams out loud that these two implementations are different.

I'm merging this anyway and gonna make slight changes on top in master.

Thank you!

Reply:
Thanks